Podenv build fails on `import Relude.Extra.Lens (Lens')`

Podenv build fails on `import Relude.Extra.Lens (Lens')`
Not sure if i'm building the same way as last time, using https://github.com/podenv/podenv/blob/main/docs/howtos/contribute.md .

```cabal
Building     dhall-1.40.1 (lib)
Installing   dhall-1.40.1 (lib)
Completed    dhall-1.40.1 (lib)
Configuring library for podenv-0.1.0.0..
Preprocessing library for podenv-0.1.0.0..
Building library for podenv-0.1.0.0..
[ 1 of 11] Compiling Paths_podenv
[ 2 of 11] Compiling Podenv.Dhall
[ 3 of 11] Compiling Podenv.Prelude

src/Podenv/Prelude.hs:49:1: error:
    Could not find module ‘Relude.Extra.Lens’
    Perhaps you meant
      Relude.Extra.Enum (from relude-0.2.0)
      Relude.Extra.Map (from relude-0.2.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
49 | import Relude.Extra.Lens (Lens')
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

<no location info>: warning: [-Wunused-packages]
    The following packages were specified via -package or -package-id flags,
    but were not needed for compilation:
      - typed-process-0.2.7.0
      - th-env-0.1.0.3
      - optparse-applicative-0.16.1.0
      - linux-capabilities-0.1.0.0
      - gitrev-1.3.1
      - SHA-1.6.4.4
cabal: Failed to build podenv-0.1.0.0 (which is required by exe:podenv from
podenv-0.1.0.0).

```

--- a/podenv.cabal
+++ b/podenv.cabal
@@ -64,7 +64,7 @@ library
                   , lens-family-th
                   , linux-capabilities
                   , optparse-applicative
-                  , relude
+                  , relude >= 0.7
                   , text
                   , th-env
                   , typed-process

